test/Microsoft.AspNet.Mvc.Core.Test/AntiXsrf/AntiForgeryWorkerTest.cs

```diff
@@ -94,7 +97,8 @@ public void ChecksSSL_GetFormInputElement_Throws()
                 serializer: null,
                 tokenStore: null,
                 generator: null,
-                validator: null);
+                validator: null,
+                htmlEncoder: null);
```

Comment:
Might want to just fill these with `new HtmlEncoder()` while we're here. Eventually this will have to pass in something  non-null.